README.md

```diff
@@ -141,7 +141,8 @@ This feature can be used to build [interactive experiences](https://warcembed-de
 | `updateUrl` | String | If provided, will replace the current `url` parameter of `<replay-web-page>`. |
 | `updateTs` | Number | If provided, will replace the current `ts` parameter of `<replay-web-page>`. |
 | `getCollInfo` | Boolean | If provided, will send a post message back with `<replay-web-page>`'s `collInfo` object, containing meta information about the currently-loaded archive. |
-| `getInited` | Boolean | If provided, will send a post message back with the current value of `<replay-web-page>`s `inited` property, indicating whether or not the service worker is ready. | 
+| `getInited` | Boolean | If provided, will send a post message back with the current value of `<replay-web-page>`s `inited` property, indicating whether or not the service worker is ready. |
+| `overrideElementAttribute` | !!Add type description here!! | If provided, will look for the element with the specified CSS selector inside `<replay-web-page>` and if found, apply the requested HTML attribute to it. If the element is not found, will send a post message back reporting `"status": "timed out"`, along with a copy of the original message's `data`. |
```

Comment:
@matteocargnelutti what do you think about here, in the README? I know that in JS the thing that I call a dictionary is not a dict but an object... I'm not sure how to shoe-horn a compound data type into this format.

Reply:
- If you end up using a `@typedef`, I'd reference it here (i.e: `HTMLAttributeOverride`, with a link to the file in which it is defined).
- Otherwise, you can use the JSDoc notation for inline objects, as such: `{{selector: string, attributeName: string, attributeContents: string}}`